bin/tests/bootstrap.bats

```diff
@@ -1,24 +1,24 @@
 #!/usr/local/bin/bats
 
-lib_dir="/usr/local/lib"
+. bin/tests/bats-common.sh
 
-load "$lib_dir/bats-support/load.bash"
-load "$lib_dir/bats-assert/load.bash"
-load "$lib_dir/bats-file/load.bash"
-
-setup () {
+function setup () {
     test_temp_dir="$(temp_make --prefix 'otomi-values-')"
     export ENV_DIR="$test_temp_dir"
     env_path="$ENV_DIR/env"
 }
 
-teardown () {
+function teardown () {
     temp_del "$test_temp_dir"
     unset ENV_DIR
     unset env_path
 }
+#######################
+# env folder creation #
+#######################
+env_folder="env folder"
 
-@test "the env folder should not be overwritten" {
+@test "$env_folder should not be overwritten" {
```

Comment:
why use a variable here? Please just use a meaningful word like `\$ENV_DIR/env`

Reply:
It is to avoid duplication, this test starts with the same name as the other one -- easy to change.